pytket/pytket/circuit/__init__.py

```diff
@@ -66,15 +68,28 @@ def overload_add_wasm(
     list_i: List[List[int]],
     list_o: List[List[int]],
     args: Union[List[int], List[Bit]],
+    **kwargs: KwargTypes,
 ) -> Circuit:
     """Add a classical function call from a wasm file to the circuit.
     \n\n:param funcname: name of the function that is called
     \n:param filehandler: wasm file handler to identify the wasm file
     \n:param list_i: list of the number of bits in the input variables
     \n:param list_o: list of the number of bits in the output variables
     \n:param args: vector of circuit bits the wasm op should be added to
+    \n:param kwargs: additional information to make the wasm op conditional
     \n:return: the new :py:class:`Circuit`"""
 
+    if kwargs is not None:
+        if "condition" in kwargs:
+            return self._add_wasm(
+                funcname,
+                str(filehandler),
+                list_i,
+                list_o,
+                args,
+                condition=kwargs["condition"],
```

Comment:
Just pass through?
```suggestion
                **kwargs
```

Reply:
I had forgotten the trick with the **, thanks for the reminder!, done in the commit.